contracts/controller/src/borrows_supplies.rs

```diff
@@ -19,16 +19,20 @@ impl Contract {
         account_id: AccountId,
         token_address: AccountId,
         token_amount: WBalance,
+        is_collateralized: bool,
```

Comment:
@dmytro-shash in such an approach, anyone could make undercollateralized borrows (which is a critical breach in the protocol).
Also, once something is live interface changes are very unwelcome.

The only thing we actually need is to allow some specific account to make such borrows.